server/aws-lsp-codewhisperer/src/language-server/codeWhispererSecurityScanServer.ts

```diff
@@ -47,21 +47,25 @@ export const SecurityScanServerToken =
             }
             try {
                 if (!credentialsProvider.hasCredentials('bearer')) {
-                    throw new Error('credentialsProvider does not have bearer token credentials')
+                    throw new Error('Credentials provider does not have bearer token credentials')
                 }
                 if (!params.arguments || params.arguments.length === 0) {
-                    throw new Error(`Incorrect params provided. Params: ${params}`)
+                    throw new Error(`Error: Incorrect parameters parameters provided`)
                 }
                 const [arg] = params.arguments
                 const { ActiveFilePath: activeFilePath, ProjectPath: projectPath } = parseJson(arg)
-                if (!activeFilePath || !projectPath) {
-                    throw new Error(`Error: file path or project path not provided. Params: ${params}`)
+                if (!activeFilePath) {
```

Comment:
In each of these failure cases, it would be good to log the params payload. As the client, we will want to diagnose by understanding what we did or didn't send over to the server.